isis/tests/FunctionalTestsFindFeatures.cpp

```diff
@@ -0,0 +1,215 @@
+#include "findfeatures.h"
+
+#include <QTemporaryFile>
+#include <QTextStream>
+#include <QStringList>
+
+#include "Fixtures.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "SurfacePoint.h"
+#include "ControlPoint.h"
+#include "Latitude.h"
+#include "Longitude.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+using ::testing::HasSubstr;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/findfeatures.xml").expanded();
+
+TEST_F(ThreeImageNetwork, FunctionalTestFindFeaturesDefault) {
+  // Setup output file
+  QVector<QString> args = {"algorithm=brisk/brisk",
+                           "match=" + tempDir.path() + "/cube3.cub",
+                           "fromlist=" + twoCubeListFile,
+                           "tolist=" + tempDir.path() + "/toList.txt",
+                           "tonotmatched=" + tempDir.path() + "/unmatched.txt",
+                           "maxpoints=5000",
+                           "epitolerance=1.0",
+                           "ratio=.65",
+                           "hmgtolerance=3.0",
+                           "onet=" + tempDir.path() + "/network.net",
+                           "networkid=new",
+                           "pointid=test_network_????",
+                           "description=new",
+                           "debug=false"};
+  UserInterface options(APP_XML, args);
+  Pvl log;
+
+  findfeatures(options, &log);
+  ControlNet network(options.GetFileName("ONET"));
+
+  ASSERT_EQ(network.GetNetworkId(), "new");
+  ASSERT_EQ(network.Description().toStdString(), "brisk/brisk/BFMatcher@NormType:NORM_HAMMING@CrossCheck:false");
+  ASSERT_EQ(network.GetNumPoints(), 50);
+}
+
+
+TEST_F(ThreeImageNetwork, FunctionalTestFindFeaturesGeomMatch) {
+  // Setup output file
+  QVector<QString> args = {"algorithm=brisk/brisk",
+                           "match=" + tempDir.path() + "/cube3.cub",
+                           "fromlist=" + twoCubeListFile,
+                           "tolist=" + tempDir.path() + "/toList.txt",
+                           "tonotmatched=" + tempDir.path() + "/unmatched.txt",
+                           "maxpoints=5000",
+                           "epitolerance=1.0",
+                           "ratio=.65",
+                           "hmgtolerance=3.0",
+                           "onet=" + tempDir.path() + "/network.net",
+                           "networkid=new",
+                           "pointid=test_network_????",
+                           "description=new",
+                           "geomsource=match",
+                           "debug=false"};
+  UserInterface options(APP_XML, args);
+  Pvl log;
+
+  findfeatures(options, &log);
+  ControlNet network(options.GetFileName("ONET"));
+  ControlPoint *pt = network.GetPoint("test_network_0001");
+  SurfacePoint sp = pt->GetAdjustedSurfacePoint();
+  Latitude lat = sp.GetLatitude();
+  Longitude lon = sp.GetLongitude();
+  ASSERT_EQ(lat.toString(), "");
+  ASSERT_EQ(lon.toString(), "");
```

Comment:
Kinda weird that these are blank but we're kinda assuming it's in a working state

Reply:
The input cubes (fixtures) didn't have lat/lon information, and since we're getting geometries from those input cubes, it left them blank.  Not sure if (or how) to add geometries to the fixtures.

Reply:
I think you would need a cube with a Mapping group, but I could be completely wrong. The projTestCube in the defaultCube fixture has one.